generic/tests/netperf.py

```diff
@@ -521,21 +533,27 @@ def get_state():
             state_list.append('intr')
             state_list.append(ninit)
 
-        io_exit = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/io_exits"))
-        irq_inj = int(
-            ssh_cmd(host, "cat /sys/kernel/debug/kvm/irq_injections"))
-        state_list.append('io_exit')
-        state_list.append(io_exit)
-        state_list.append('irq_inj')
-        state_list.append(irq_inj)
+        if arch.ARCH in ('ppc64', 'ppc64le'):
+            exits = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/exits"))
+            state_list.append('exits')
+            state_list.append(exits)
+        else:
+            io_exits = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/io_exits"))
+            irq_injs = int(
+                ssh_cmd(host, "cat /sys/kernel/debug/kvm/irq_injections"))
+            state_list.append('io_exits')
+            state_list.append(io_exits)
+            state_list.append('irq_injs')
+            state_list.append(irq_injs)
+
         return state_list
 
     def netperf_thread(i, numa_enable, client_s, timeout):
         cmd = ""
         fname = "/tmp/netperf.%s.nf" % pid
         if numa_enable:
-            output = ssh_cmd(client_s, "numactl --hardware")
-            n = int(re.findall(r"available: (\d+) nodes", output)[0]) - 1
+            output = ssh_cmd(client_s, "numactl --show|grep nodebind")
```

Comment:
It looks use command 'numactl --hardware' to find a available node looks more reasonable.